assets/js/components/NewChecksResults/CheckResultOutline.jsx

```diff
@@ -0,0 +1,125 @@
+import React from 'react';
+import {
+  EOS_COLLOCATION,
+  EOS_DESKTOP_WINDOWS,
+  EOS_KEYBOARD_ARROW_RIGHT,
+} from 'eos-icons-react';
+import classNames from 'classnames';
+
+const TARGET_NODE = 'TARGET_NODE';
+const TARGET_CLUSTER = 'TARGET_CLUSTER';
+
+const isExpect = ({ type }) => type === 'expect';
+const isExpectSame = ({ type }) => type === 'expect_same';
+
+function CheckResultOutline({
+  checkId,
+  agentsCheckResults,
+  expectationResults,
+  clusterName,
+}) {
+  const expectExpectationsCount = expectationResults.filter(isExpect).length;
+  const expectResults = agentsCheckResults.map((agentCheckResult) => {
+    const { hostname, expectation_evaluations } = agentCheckResult;
+
+    const isError = !!agentCheckResult?.type;
```

Comment:
Maybe we could wrap this tiny `!!agentCheckResult?.type;` somehow as `isAgentCheckError` or similar.
At least, to me personally, it would help to know that we are dealing with a `AgentCheckError` object type and not a `AgentCheckResult` object

Reply:
changed to `const isAgentCheckError = !!agentCheckResult?.type;`